### DISCUSSION TITLE
What are details and deadline for V3?

### DISCUSSION THREAD

**Discussion by experdot** [elsa-workflows/elsa-core#1091] [Design & tech]
The list below is too simple. Could you provide a more concrete planning?

- [ ]  Composite Activity Definitions (with designer support)
- [ ]  Job Activities (simplify starting background processes while the workflow sleeps & gets resumed once the job finishes)
- [ ]  Lucene Indexing
- [ ]  Localization Support
- [ ]  State Machines
- [ ]  Sagas

**Comment by sfmskywalker** [comment]
I don't have any concrete planning at the moment, just some rough ideas on what I want to see in V3. However, more details will be added over time to the [Elsa 3.0 milestone](https://github.com/elsa-workflows/elsa-core/milestone/9).

If there's anything you'd like to see for V3, please do let me know!
